src/Plugin.php

```diff
@@ -64,6 +64,8 @@ public function bootstrap(PluginApplicationInterface $app): void
      */
     public function console(CommandCollection $commands): CommandCollection
     {
-        return $commands->add('worker', WorkerCommand::class);
+        return $commands
+            ->add('queue work', WorkerCommand::class)
+            ->add('worker', WorkerCommand::class);
```

Comment:
@markstory Should we just stick with the new convention and remove this in 1.0?

Reply:
I'm fine with that. Aliases are low cost to maintain until we get to 1.0.